src/features/defi/components/LearnMore/LearnMore.tsx

```diff
@@ -0,0 +1,109 @@
+import { ArrowBackIcon } from '@chakra-ui/icons'
+import { Box, Flex } from '@chakra-ui/layout'
+import { Button, IconButton } from '@chakra-ui/react'
+import { useSteps } from 'chakra-ui-steps'
+import osmosis from 'assets/osmosis.svg'
+import { CarouselDots } from 'components/CarouselDots/CarouselDots'
+import { SlideTransition } from 'components/SlideTransition'
+import { Text } from 'components/Text'
+
+import { DefiModalHeader } from '../DefiModalHeader/DefiModalHeader'
+
+const STEPS_LENGTH = 3
+const STEP_TO_ELEMENTS_MAPPING = {
```

Comment:
The rationale to this mapping vs. having that on the translations const is because it keeps things more declarative and clean there